torch/storage.py

```diff
@@ -29,7 +29,6 @@ class _StorageBase:
     def __init__(self, *args, **kwargs): ...  # noqa: E704
     def __len__(self) -> int: ...  # noqa: E704
     def __getitem__(self, idx): ...  # noqa: E704
-    def __setitem__(self, *args, **kwargs): ...  # noqa: E704
```

Comment:
It is intentionally. This code was introduced by https://github.com/pytorch/pytorch/pull/94503. This PR drops the code added by #94503.

So, since no one uses this `self[] = ...` now, I deleted this. Of course, we can still keep this. Which is preferable, keep or delete?